src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -37,11 +37,15 @@ class OgSelection extends DefaultSelection {
    * @return DefaultSelection
    */
   public function getSelectionHandler() {
-    $field = $this->getConfiguration('field');
-
-    if ($field) {
-      return \Drupal::service('plugin.manager.entity_reference_selection')->getSelectionHandler($field);
```

Comment:
See below comment about the plugin IDs, the OG selection handler was never being used for group selection, until now. So you might need to update those to see those problems locally. Otherwise it will continue to use the node selection handler.